protocol/app/app.go

```diff
@@ -1242,6 +1243,15 @@ func (app *App) BeginBlocker(ctx sdk.Context, req abci.RequestBeginBlock) abci.R
 		)
 	}
 
+	// Log out app version every hour.
+	if ctx.BlockTime().Unix()%3600 == 0 {
```

Comment:
If we're stuck on a block, will this `BlockTime` still update? 

Also, sorry I mispoke but we should provide telemetry on this rather than a log (or maybe both? but put them in the same spot)

Reply:
oops didn't realize they were already in the same spot